apps/dashboard/src/@3rdweb-sdk/react/hooks/useApi.ts

```diff
@@ -506,27 +505,25 @@ export function useUpdateAccountPlan(waitForWebhook?: boolean) {
 
       return json.data;
     },
-    {
-      onSuccess: async () => {
-        // invalidate usage data as limits are different
-        await queryClient.invalidateQueries({
-          queryKey: accountKeys.me(user?.address as string),
-        });
+    onSuccess: async () => {
+      // invalidate usage data as limits are different
+      await queryClient.invalidateQueries({
+        queryKey: accountKeys.me(user?.address as string),
+      });
 
-        return queryClient.invalidateQueries({
-          queryKey: accountKeys.usage(user?.address as string),
-        });
-      },
+      return queryClient.invalidateQueries({
+        queryKey: accountKeys.usage(user?.address as string),
+      });
```

Comment:
put these in a `promise.all` and return that

Reply:
Updated